pkg/interactor/data.go

```diff
@@ -45,12 +46,12 @@ func (d *DataInteractor) Create(data Data) error {
 }
 
 func (d *DataInteractor) DeleteByDevice(deviceId string) (error error) {
-	err := dataStore.DeleteByDevice(deviceId)
+	err := dataStore.Delete(bson.M{"from": deviceId})
```

Comment:
The interactor layer shouldn't know about database details (the `bson` format, in this case). Passing `deviceId` is completely fine.

Reply:
Done!